pg.gemspec

```diff
@@ -23,7 +23,9 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features|translation)/}) }
+    `git ls-files -z`.split("\x0").reject do |f|
+      f.match(%r{\A(?:test|spec|features|translation|\.github|\.travis|\.appveyor)})
```

Comment:
🤔 Perhaps all dot-files (beginning with `.`) should be rejected?

Reply:
Interesting point. I can change to this.

BTW, specifying a list of file is better than "all, but exclude some". Yeah, it can lead to a release error, when you've added new files and forgot to list them in `gemspec`, but it's still more accurate and secure.

But I just wanted to make a small patch instead of discussing an opposite approach.

Reply:
Done. I'm OK with both approaches (list CI files or all dot-files), but, again, in the future it's better to list required files only.